website/docs/advanced-concepts/access-control/roles.md

```diff
@@ -0,0 +1,247 @@
+
+
+# Roles
+ A Role is a collection of permissions that grant access to perform certain operations on resources and can be assigned to a user or a group. Roles enable users to manage multiple permissions rather than assigning individual permissions to users/groups. In Appsmith, a role can be mapped to multiple permissions and a permission can be mapped to multiple roles. However, a role cannot be nested, i.e. it can contain only permissions and not other roles. Roles are useful for efficiently managing permissions to access Appsmith resources.
+ 
+ Appsmith provides three built-in roles - **Admin**, **Developer** and **App Viewer**.  With GAC, you can create a custom role that provides fine-grained access control. 
```

Comment:
should we rename **Admin** to **Administrator**?
<img width="645" alt="image" src="https://user-images.githubusercontent.com/28362912/209928675-832b5edf-c023-4e3f-adf0-142ee86d1980.png">


Also, should we write the full form of GAC the first time we are using it and then mention its short form in bracket?